packages/veto-app/shared/db/src/index.ts

```diff
@@ -0,0 +1,12 @@
+const gun = require('./gun')
```

Comment:
Leftover `require`, convert it to `import` :)

Reply:
I have a TS error if I use `import`: 
```
File 'zenful/veto/packages/veto-app/shared/db/src/gun/index.ts' is not a module.
```

Reply:
On front-end or back-end?

Using require only obfuscate the problem, we should fix this.

Reply:
I had similar problems with the front-end when the library wasn't built before I ran the front-end.

Try using import, but make sure veto-app/shared/db is fully built (check the out/ directory). A good trick is to add a `prebuild` step where you delete the out folder (like we do here https://github.com/Veriphi/veto/blob/master/packages/maintenance-app/types/package.json#L12). It ensure we fully rebuild the library every time.